spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/spanner/GcpSpannerAutoConfigurationTests.java

```diff
@@ -38,7 +37,8 @@
 
 	private ApplicationContextRunner contextRunner = new ApplicationContextRunner()
 			.withConfiguration(AutoConfigurations.of(GcpSpannerAutoConfiguration.class,
-					GcpContextAutoConfiguration.class))
+					GcpContextAutoConfiguration.class,
+					SpannerRepositoriesAutoConfiguration.class))
```

Comment:
@artembilan do you know why this test is failing with:

```
Caused by: java.lang.IllegalStateException: Unable to retrieve @EnableAutoConfiguration base packages
	at org.springframework.boot.autoconfigure.AutoConfigurationPackages.get(AutoConfigurationPackages.java:76)
	at org.springframework.boot.autoconfigure.data.AbstractRepositoryConfigurationSourceSupport.getBasePackages(AbstractRepositoryConfigurationSourceSupport.java:79)
```

I can't add `@EnableAutoConfiguration` to `SpannerRepositoriesAutoConfiguration` since it will enable every other autoconfiguration in the package.
I added `SpannerRepositoriesAutoConfiguration` to `spring.factories`, shouldn't that be enough?